### DISCUSSION TITLE
[helm] Specified S3 bucket does not exist, running the launcher replication-orchestrator failed

### DISCUSSION THREAD

**Discussion by rob-teeuwen** [airbytehq/airbyte#33100] [Connector Questions] [answered]
I installed Airbyte on a managed Azure kubernetes cluster using the default helm chart. I didn't change anything. I'm using Airbyte 0.50.35. 

I set up 2 simple connections (mysql > motherduck, and mysql > mysql). Both connections fail to sync with the same error message: `io.temporal.serviceclient.CheckedExceptionWrapper: io.airbyte.workers.exception.WorkerException: Running the launcher replication-orchestrator failed`. Upon inspection of the logs in the worker pod, I'm seeing something weird: 

```
Caused by: io.temporal.failure.ApplicationFailure: message='The specified bucket does not exist 
(Service: S3, Status Code: 404, Request ID: 179DF8F3401D6D19, Extended Request ID: 
dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8)', 
type='software.amazon.awssdk.services.s3.model.NoSuchBucketException', nonRetryable=false
```

I'm thinking this is weird because I'm on Azure, I don't use S3. So I'm not sure why it's trying to connect to an S3 bucket here. It seems to be similar as this unanswered question: https://github.com/airbytehq/airbyte/discussions/32998

**Comment by sajarin** [comment]
hey @rob-teeuwen are you using any cloud logging with your Airbyte instance?

**Reply by rob-teeuwen** [reply]
Not that I know, I deployed it completely default using the official helm chart without overriding any values.

**Comment by marcosmarxm** [comment, marked as answer]
@rob-teeuwen this is a problem with Airbyte not creating one particular minio bucket. Check [workaround](https://github.com/airbytehq/airbyte/issues/32203#issuecomment-1832627982). I'll close this as duplicate from the other issues. It doesn't have anything related to S3 :) unfortunately logs aren't clear.

**Reply by rob-teeuwen** [reply]
Great, thank you!
